ansible_collections/arista/avd/docs/contribution/overview.md

```diff
@@ -65,100 +65,100 @@ Should this happen, and if you followed the project guidelines, have sample test
-    * `Reformat`: Refactor of formatting, for example, omit whitespace.
-    * `Optimize`: Refactor of performance, for example, speed up code.
-    * `License`: Edits regarding licensing; no production code change.
-    * `Revert`: Change back to the previous commit.
+    - `Feat`: Create a capability, for example, feature, test, or dependency.
+    - `Fix`: Fix an issue, for example, bug, typo, accident, or misstatement.
+    - `Cut`: Remove a capability, for example,  feature, test, or dependency.
+    - `Doc`: Refactor documentation, for example, help files.
+    - `CI`: Update CI components, for example,  molecule files or GitHub Actions.
+    - `Start`: Begin doing something, for example, create a feature flag.
+    - `Stop`: End doing something, for example, remove a feature flag.
+    - `Bump`: Increase the version of something (dependency).
+    - `Test`: Add or refactor anything regarding a test, for example, add a new test case.
+    - `Make`: Change the build process, tooling, or infrastructure.
+    - `Refactor`: A code change that only contains refactoring.
+    - `Reformat`: Refactor of formatting, for example, omit whitespace.
+    - `Optimize`: Refactor of performance, for example, speed up code.
+    - `License`: Edits regarding licensing; no production code change.
+    - `Revert`: Change back to the previous commit.
 
-  * **Scopes**:
+  - **Scopes**:
 
-    * `{{ role_name }}`: AVD role impacted by PR. **Required** for `Feat`, `Cut` and `Fix` types.
-    * `plugins`: To use when AVD plugin is impacted by PR. **Required** for `Feat`, `Cut` and `Fix` types.
-    * `requirements`: To use when using `Bump` type and when any AVD requirement is updated.
-    * `mkdoc`: Represents generic documentation published on [avd.sh](https://avd.sh/en/stable/).
-    * `contribution`: Documentation related to contribution.
-    * `how-to`: Documentation in the how-to section of AVD.
-    * `actions`: Represents GitHub Actions update.
-    * `molecule`: Represents Molecule CI update.
-    * `ansible`: For any Ansible information update like `galaxy.yml` or Ansible requirements.
-    * `github`: For any content related to GitHub processes.
+    - `{{ role_name }}`: AVD role impacted by PR. **Required** for `Feat`, `Cut` and `Fix` types.
+    - `plugins`: To use when AVD plugin is impacted by PR. **Required** for `Feat`, `Cut` and `Fix` types.
+    - `requirements`: To use when using `Bump` type and when any AVD requirement is updated.
+    - `mkdoc`: Represents generic documentation published on [avd.sh](https://avd.sh/en/stable/).
+    - `contribution`: Documentation related to contribution.
+    - `how-to`: Documentation in the how-to section of AVD.
+    - `actions`: Represents GitHub Actions update.
+    - `molecule`: Represents Molecule CI update.
+    - `ansible`: For any Ansible information update like `galaxy.yml` or Ansible requirements.
+    - `github`: For any content related to GitHub processes.
```

Comment:
@ClausHolbechArista, @JulioPDX  this needs to be adapted to what we agreed upon in #1930 - so less types and less scopes

https://github.com/aristanetworks/ansible-avd/pull/1930#issuecomment-1216413844

Reply:
addressed in #1930